opencti-platform/opencti-front/src/private/components/workspaces/WorkspacePopover.jsx

```diff
@@ -122,12 +122,10 @@ const WorkspacePopover = ({ workspace, paginationOptions }) => {
       </IconButton>
       <Menu anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={handleClose}>
         <MenuItem onClick={handleOpenEdit}>{t_i18n('Update')}</MenuItem>
-        {workspace.type === 'dashboard' && (
-        <>
-          <MenuItem onClick={handleDashboardDuplication}>{t_i18n('Duplicate')}</MenuItem>
-          <MenuItem onClick={() => handleExportJson(workspace)}>{t_i18n('Export')}</MenuItem>
-        </>
-        )}
+        {workspace.type === 'dashboard' && [
+          <MenuItem key="menu_duplicate" onClick={handleDashboardDuplication}>{t_i18n('Duplicate')}</MenuItem>,
+          <MenuItem key="menu_export" onClick={() => handleExportJson(workspace)}>{t_i18n('Export')}</MenuItem>,
+        ]}
```

Comment:
is that ok to have component in a list?

Reply:
MUI's Menu seem to work this way, judging from the error it threw before:

```
MUI: The Menu component doesn't accept a Fragment as a child.
Consider providing an array instead. 
```